### PR TITLE
[PB-1525] feat/persist the language of the website in drive-web

### PR DESCRIPTION
**Pull request by rafijv**
- Set the language of the website in drive-web using query-params or coockies
- Add and fix some translations

### CODE REVIEW

File: src/app/core/views/Preferences/tabs/Account/Language.tsx [hunk 1]
```diff
@@ -10,9 +10,9 @@ import dayjs from 'dayjs';
 import ItemsDropdown from './components/ItemsDropdown';
 import MenuItem from './components/MenuItem';
 
-const localStorageLanguage = localStorageService.get('language');
+const localStorageLanguage = localStorageService.get('i18nextLng');
 
-const languages = ['en', 'es', 'fr', 'it', 'cn', 'ru', 'de'];
+const languages = ['en', 'es', 'fr', 'it', 'zh', 'ru', 'de'];
```
Comment (CandelR) [inline src/app/core/views/Preferences/tabs/Account/Language.tsx:15]: also need to change `types.ts` in i18n folder

File: src/app/core/views/Preferences/tabs/Account/InviteAFriend.tsx [hunk 2]
```diff
@@ -37,7 +37,7 @@ export default function InviteAFriend({ className = '' }: { className?: string }
       const castedError = errorService.castError(err);
       if (castedError.message !== 'Mail invitation daily limit reached') {
         notificationsService.show({
-          text: translate('error.inviteAFriend', { message: castedError.message }),
+          text: translate('error.inviteAFriendEmpty'),
```
Comment (CandelR) [inline src/app/core/views/Preferences/tabs/Account/InviteAFriend.tsx:40]: If `inviteAFriend` translation is not used any more, please, remove it

**Review by CandelR — CHANGES_REQUESTED** [review]

File: src/app/core/views/Preferences/tabs/Account/Language.tsx (hunk 1 above)
Comment (rafijv) [reply src/app/core/views/Preferences/tabs/Account/Language.tsx:15]: Done

File: src/app/core/views/Preferences/tabs/Account/InviteAFriend.tsx (hunk 2 above)
Comment (rafijv) [reply src/app/core/views/Preferences/tabs/Account/InviteAFriend.tsx:40]: It is used in another component

**Review by CandelR — APPROVED** [review]

**Pull request merged**